### PR TITLE
Removed some sneaky bootsrap all import, moved to grid for about page

### PR DESCRIPTION
**Pull request by lili2311**
@womenhackfornonprofits/whfnp
## What are the changes included in this PR?

Moved about us page to bootstrap grid to fix the centering issue.
Removed a bootstrap all import to allow for only select imports.
## If the changes are visual, provide a screenshot.

download 1
download

### CODE REVIEW

File: src/sass/screen.scss [hunk 1]
```diff
@@ -271,17 +273,17 @@ h3 {
...
-	margin: 0 auto;
-	padding: 0;
-	@media (min-width: 600px) {
-		padding:0;
-		.col-3 {
-			min-height: 15em;
-		}
-	}
-}
+// .founding-team, .non-profits-wrap, .volunteers-wrap{
```
Comment (EChesters) [inline src/sass/screen.scss]: Are we better off removing this code if it's being commented out and not breaking anything or being used anywhere?

File: src/about.html [hunk 2]
```diff
@@ -21,14 +21,14 @@ <h2 class="section-heading">What We Do</h2>
...
-			<div class="col-2">
+		<div class="row">
+			<div class="col-xs-12 col-sm-6">
 				<a href="https://uk.linkedin.com/in/vinitarathi" class="volunteer-link">
 					<img class="volunteer-picture" src="img/volunteers/Vinita_Rathi.png" alt="Vinita Rathi Picture">
 					<h2>Vinita Rathi</h2>
 				</a>
 			</div>
-			<div class="col-2">
+			<div class="col-xs-12 col-sm-6">
```
Comment (EChesters) [inline src/about.html:31]: If we have different sizes being introduced, can we have a screenshot for each, of the whole page rather than a snippet?

Comment (lili2311) [inline src/about.html:31]: added screenshots of both mobile and desktop

**Comment by lili2311** [comment]
Since all comments have been addresses im merging this in.

**Pull request merged**